Modules/_sqlite/row.c

```diff
@@ -96,6 +96,25 @@ PyObject* pysqlite_row_item(pysqlite_Row* self, Py_ssize_t idx)
    return Py_XNewRef(item);
 }
 
+static int
+pysqlite_row_contains(pysqlite_Row* self, PyObject *arg)
+{
+    Py_ssize_t nitems, i;
+    int cmp = 0;
+
+    nitems = PyTuple_Size(self->description);
+
+    for (i = 0; cmp == 0 && i < nitems; i++) {
+        cmp = PyObject_RichCompareBool(
+            arg,
+            PyTuple_GET_ITEM(PyTuple_GET_ITEM(self->description, i), 0),
```

Comment:
There's a lot happening on this line. I'd prefer it if you extracted these items before calling `PyObject_RichCompareBool`, for improved readability.